go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/docker/distribution
+module github.com/distribution/distribution/v3
 
 go 1.12
```

Comment:
Should we bump this to `1.13` at least? Or ideally, match it with what's in `travis.yaml` i.e. `1.14`?

Reply:
I was considering changing it, but it's not directly related to this change. Generally I try to be a conservative to allow projects to build with other Golang versions (although Golang does not (yet) "enforce" the go version, [it's possible they start doing so](https://github.com/golang/go/issues/30446#issuecomment-468044008)).

Reply:
This should probably reflect the minimum required version to build the project. I don't think we rely in any 1.13+ feature, so seems OK to keep it _as-is_.

Reply:
Yes; and even if building the "whole" project requires Go version X, it's possible there are consumers that only use a single package from this repo, which may in itself not require that version; we could _document_ that the project was "tested on" / "requires" / "recommends" a specific Go version to build (e.g., in "BUILDING.md").

Reply:
👍 Sounds good. I was thinking more along the lines of "dropping the support for older versions of Go", but this is ok with me. Adding `BUILD.md` file with notes and instructions sounds good.